frontend/integration-tests/views/metalkube/dashboards.view.ts

```diff
@@ -0,0 +1,127 @@
+import { browser, element, by , $$ } from 'protractor';
+import { waitForNone } from '../../protractor.conf';
+
+export const untilNoLoadersPresent = waitForNone($$('.co-m-loader'));
+export const isLoaded = () => browser.wait(untilNoLoadersPresent).then(() => browser.sleep(2000));
+
+// Usage example:
+// await dashboardView.inventoryItemLabel(0).getText()
+export function inventoryItemLabel(rowNumber) {
+  return element(by.xpath(`(//div[@class="kubevirt-inventory__row-title"])[${rowNumber + 1}]`));
+}
+
+// Usage example:
+// await dashboardView.inventoryUpCounter(0).getText()
+export function inventoryUpCounter(rowNumber) {
+  // Explaining the xpath:
+  // There is an array of 6 kubevirt-inventory__row-status items (one for each line in the inventory).
+  // Then we look for a kubevirt-inventory__row-status-item which has a kubevirt-inventory__row-status-item-icon--ok
+  // under it, and go __back up__ to the parent (with "..") and find the span under it. This gives us
+  // the counter *next to* the kubevirt-inventory__row-status-item-icon--ok.
+  return element(by.xpath(`(//div[@class="kubevirt-inventory__row-status"])[${rowNumber + 1}]` +
+    '/div[contains(@class, "kubevirt-inventory__row-status-item")]' +
+    '/span[contains(@class, "kubevirt-inventory__row-status-item-icon--ok")]' +
+    '/../span[contains(@class, "kubevirt-inventory__row-status-item-text")]'));
+}
+
+// Usage example:
+// await dashboardView.inventoryDownCounter(0).getText()
+export function inventoryDownCounter(rowNumber) {
+  // Similar xpath explanation to the one in inventoryUpCounter(), but now we're finding the counter
+  // next to the kubevirt-inventory__row-status-item-icon--error.
+  return element(by.xpath(`(//div[@class="kubevirt-inventory__row-status"])[${rowNumber + 1}]` +
+    '/div[contains(@class, "kubevirt-inventory__row-status-item")]' +
+    '/span[contains(@class, "kubevirt-inventory__row-status-item-icon--error")]' +
+    '/../span[contains(@class, "kubevirt-inventory__row-status-item-text")]'));
+}
+
+// substrings to identify the rows in the inventory card
+export const INVENTORY_NODES = ' Nodes';
+export const INVENTORY_HOSTS = ' Hosts';
+export const INVENTORY_PVCS = ' PVCs';
+export const INVENTORY_PODS = ' Pods';
+export const INVENTORY_VMS = ' VMs';
+export const INVENTORY_DISKS = ' Disks';
+
+// Usage example:
+// const rowNumber = await dashboardView.inventoryRow(dashboardView.INVENTORY_HOSTS);
+export const inventoryRow = async(substrInventory) => {
```

Comment:
Have you tried the method I suggested? Doesn't it work for you?

Reply:
I looked at the method you suggested, but decided to stay with the code I have because the alternatives are more or less the same complexity level.

Reply:
An ID or css class on the counters can help but there is still a "gotcha": we'll get an array of elements, but won't know which one is in which row because not all counters are always displayed (counters equal to 0 are simply not rendered).